src/e2e/specs/landing.spec.ts

```diff
@@ -90,30 +85,19 @@ test.describe("Landing Page Functionality Verification", () => {
         "https://testrail.stage.mozaws.net/index.php?/cases/view/2255913",
     });
 
-    // Intercept request and fulfill return
-    await page.route("**/api/v1/scan/", async (route) => {
-      await route.fulfill({
-        status: 200,
-        body: JSON.stringify({
-          success: true,
-          breaches: [],
-          total: 0,
-          heading:
-            'We found (mocked) <span class="breach-result-email"></span> exposed in <span class="breach-result-count"></span> data breaches.',
-          dataClassStrings: [],
-          logos: [],
-        }),
-      });
+    await page.goto(process.env.E2E_TEST_BASE_URL as string, {
+      waitUntil: "networkidle",
     });
 
     // generate email
     const randomEmail = `${Date.now()}_auto@restmail.net`;
 
-    // await page.pause()
+    // enter email for free scan
     await landingPage.enterScanEmail(randomEmail);
 
-    // verify scan page items
-    await expect(async () => {
+    const freeScanRedirect = await waitForUrlOrTimeout(page, "/scan", 5000);
```

Comment:
If this is necessary, shouldn't it be part of `enterScanEmail`?

Reply:
good point, will have the result be returned in enterFreeScanEmail function